### PR TITLE
Feature/challenge

### PR DESCRIPTION
**Pull request by rabiul042**
solved challenge form validation issue both existing user and email.

### CODE REVIEW

File: app/server/formSchemas/challenge.js [hunk 1]
```diff
@@ -2,47 +2,6 @@ export default {
...
-      propertyOrder: 4
-    },
-    challenged: {
-      type: 'hidden',
-      title: null,
-      widget: 'hidden',
-      icon: null,
-      help: null,
-      propertyOrder: 5
-    },
```
Comment (mahmed0715) [inline app/server/formSchemas/challenge.js]: Why should we remove schema?

File: app/server/formSchemas/registerTrees.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ export default {
         propertyOrder: 8
       }
     },
-    required: ['treeCount', 'plantDate'],
```
Comment (mahmed0715) [inline app/server/formSchemas/registerTrees.js]: Why its changed register trees?

File: app/server/formSchemas/registerTrees.js [hunk 3]
```diff
@@ -203,7 +203,7 @@ export default {
         propertyOrder: 5
       }
     },
-    required: ['treeCount', 'plantDate'],
```
Comment (mahmed0715) [inline app/server/formSchemas/registerTrees.js]: Looks like the new branch is not created from latest develop branch.

**Comment by rabiul042** [comment]
@mahmed0715 Thanks for the advice. I'll follow this.

**Comment by mahmed0715** [comment]
Duplicate https://github.com/Plant-for-the-Planet-org/treecounter-app/pull/1647

**Pull request closed without merging**